ui/components/app/assets/nfts/nft-grid/nft-grid-item-error-boundary.tsx

```diff
@@ -0,0 +1,38 @@
+import log from 'loglevel';
+import { Component, ReactNode, ErrorInfo } from 'react';
+
+type ErrorBoundaryProps = { children: ReactNode; fallback: () => ReactNode };
+type ErrorBoundaryState = { hasError: boolean };
+
+class NFTGridItemErrorBoundary extends Component<
```

Comment:
Sad that error boundaries like this need to be class components, seems like a framework limitation of React.